cmd/erasure-server-pool-decom.go

```diff
@@ -705,7 +705,11 @@ func (z *erasureServerPools) decommissionPool(ctx context.Context, idx int, pool
 				}
 				var failure bool
 				// gr.Close() is ensured by decommissionObject().
-				if err = z.decommissionObject(ctx, bName, gr); err != nil {
+				for try := 0; try < 3; try++ {
+					if err = z.decommissionObject(ctx, bName, gr); err == nil {
+						failure = false
+						break
+					}
```

Comment:
This retry will not work since the `gr` will be closed and the subsequent reads will fail @krishnasrinivas